src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  background-color: rgb(222, 219, 219);
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  min-width: 1024px;
+  padding: 0;
+  margin: 0;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order